.phpstorm.meta.php

```diff
@@ -6,6 +6,9 @@
 // $container->get(Type::class) → instance of "Type"
 override(\Psr\Container\ContainerInterface::get(0), type(0));
 
+// $propertyPhpDocInfo->getByType(Type::class) → instance of "Type"|null - @todo how to make this nullable?
+override(\Rector\BetterPhpDocParser\PhpDocInfo\PhpDocInfo::getByType(0), type(0));
```

Comment:
@nikic Do you have any tip how to teach PHPStorm nullable type?

I tried variations of ↓... but no success

```diff
-type(0)
+type(0)|null
```

Reply:
Unfortunately this isn't supported right now. PhpStorm itself doesn't do much in terms of nullability analysis, so the missing null type probably won't result in false positive warnings at least.

Reply:
I see, feature then, thank you :+1: